shenyu-examples/shenyu-examples-dubbo/shenyu-examples-spring-cloud-alibaba-dubbo-service-annotation/src/main/java/org/apache/shenyu/examples/spring/cloud/dubbo/service/annotation/impl/TestServiceImpl.java

```diff
@@ -30,7 +30,7 @@
 /**
  * The type Dubbo service.
  */
-@DubboService
+@ShenyuDubboService("test")
```

Comment:
rollback this

Reply:
Why was this not rollbacked? It seems that Dubbo only resolves the direct annotation, not composite annotation.